src/common/snippets/src/pass/assign_registers.cpp

```diff
@@ -5,6 +5,7 @@
 #include <snippets/itt.hpp>
 #include "snippets/pass/assign_registers.hpp"
 #include "snippets/snippets_isa.hpp"
+#include <iterator>
```

Comment:
why do we need this include?

Reply:
I added this include to fix Windows precommit tests. The test failed with : msvc  'inserter' is not a member of 'std'
Shall I remove it anyway?